includes/widgets/image-carousel.php

```diff
@@ -92,6 +92,10 @@ public function get_style_depends(): array {
 		return [ 'e-swiper', 'widget-image-carousel' ];
 	}
 
+	public function get_script_depends(): array {
+		return [ 'import-script-carousel-base' ];
```

Comment:
@rami-elementor Louis is correct here.

Alternatively we could use an additional function called something like `get_import_scripts`.

I opted to use the existing `get_script_depends`, because in this way the logic works in a straight forward way, and we don't need much additional infra.

Reply:
Any chance we can change this `import-script-` prefix?

Reply:
@hein-obox  Does `elementor-` sounds good?

Reply:
@rami-elementor That's fine with me.

Reply:
The short version `e-` is also good.